### DISCUSSION TITLE
Admin Panel

### DISCUSSION THREAD

**Discussion by Koltsz** [dani-garcia/vaultwarden#4084] [Q&A] [answered]
Hi,

I'm using version 1.30.0

Having an issue with the admin token.

I have generated it using the container and I've generated it on my Linux machine.
Ive tried with and without symbols

I've added the double $$ as I'm using docker compose and I have used single quotes ' '

No matter how I paste that token into the admin page it will not let me in.

What could I be doing wrong. How is it supposed to be pasted into the admin console?

**Comment by BlackDex** [comment, marked as answer]
What do the logs mention during start?
Do you have a `config.json` file in your data folder?
What if you try a simple plain text password for testing?

**Reply by Koltsz** [reply]
I don't have a config.json file,
I will give it a shot with a plain text password and go from there

**Reply by Koltsz** [reply]
That worked but shows a warning. im going to assume it is secure public facing? however to be safe i have blocked the /admin path using my proxy

![image](https://github.com/dani-garcia/vaultwarden/assets/36265128/0a0d0931-693f-4ffd-acf4-7d06904bdfa9)

**Reply by danieldotkim** [reply]
Were you able to get the config.json file to appear? Mine will not appear in my data folder.
